src/main/kotlin/uk/gov/dluhc/notificationsapi/client/GovNotifyApiClient.kt

```diff
@@ -19,22 +20,31 @@ private val logger = KotlinLogging.logger {}
 @Component
 class GovNotifyApiClient(
     private val notificationClient: NotificationClient,
-    private val sendNotificationResponseMapper: SendNotificationResponseMapper
+    private val sendNotificationResponseMapper: SendNotificationResponseMapper,
+    @Value("\${api.notify.catch-wrong-api-key-errors}") private val catchWrongApiKeyErrors: Boolean,
```

Comment:
I don't have a natural thing in my mind of what "catch" means, would ignore/swallow be a clearer description of what this does